clio-transfer-model/src/main/scala/org/broadinstitute/clio/transfer/model/TransferGvcfV1QueryInput.scala

```diff
@@ -0,0 +1,10 @@
+package org.broadinstitute.clio.transfer.model
+
+import org.broadinstitute.clio.util.model.{DocumentStatus, Location}
+
+case class TransferGvcfV1QueryInput(documentStatus: Option[DocumentStatus] =
+                                      None,
```

Comment:
Strange indentation here

Reply:
I think that's scalafmt because I keep on fixing it, and it keeps on reformatting whenever I run the tests :( But I'll make sure I fix it before committing